invenio_app_ils/circulation/api.py

```diff
@@ -38,3 +38,28 @@ def request_loan(params):
     )
 
     return pid, loan
+
+
+def create_loan(params):
```

Comment:
is there a reason why we won't reuse request_loan function above and just pass the trigger as a param?

Reply:
Because the request was for creating a pending loan request and this had different permissions.